using/solving-exercises/tdd.md

```diff
@@ -0,0 +1,78 @@
+# Test-Driven Development (TDD)
+
+Test-Driven Development (sometimes called Test-First Development or Test-Driven Design) is the practice of writing the unit tests first, before you write a single line of implementation code.
+
+## On Exercism, the tests _are_ the requirements!
+
+Each exercise you work on will have some instructions describing in general terms what you need to do.
+By design, these instructions do not account for programming-language-specific implementation details because they are shared by all of Exercism’s 70+ language tracks.
+Some language tracks will append more specific details for you, but not all of them do.
+
+When you start working on an exercise, give the instructions a careful read.
+They will give you a broad overview of how you go about implementing a solution.
+But you will have to read the _tests_ to understand the full and exact requirements:
+
+- Must the result be a particular kind of data structure?
+- Must the result be sorted in some order?
+- How are you expected to handle exceptions? And so on.
+
+You have solved an exercise when all the provided tests run and pass.
+In other words, your solution is not just an interpretation the instructions that "looks right", your solution is a program that _satisfies the given tests_.
+**The tests represent the complete requirements for the exercise.**
+
+## Rationale for Test-Driven Development
```

Comment:
I'd be tempted to move this even further, below "## Working locally" - just because I feel for 95% of people, the details they're reading this for are the skips and the "what do I do". The more I work with people the more I realise that the "why?" comes a long way after the "what is the immediate thing I need to do to satisfy myself" in their priorities, and if its not very quickly obvious, they don't read.